### DISCUSSION TITLE
bind tags together

### DISCUSSION THREAD

**Discussion by hellersqueen** [AprilSylph/XKit-Rewritten#1036] [Suggestion Box]
would it be possible to basically bind tags together? i like quick tags but i have so many bundles it's becoming difficult to get what i'm looking for. i was wondering if there could be a way to make two or more tags kinda "bound" together so that me writing one tag would immediately make the others appear too.

**Comment by marcustyphoon** [comment]
That's what a tag bundle is, no? You can put more than one tag in a bundle and when you click it, all of the tags included will be added.

**Reply by hellersqueen** [reply]
yes like i said bundles are great and all but i was wondering if there was a way this could be more of an automatic process while i'm writing the tags because finding the right bundle when i have a billion of them is getting hard. that's on me for having so many i know but i just thought i'd ask

**Reply by marcustyphoon** [reply]
Sorry, I don't think I get what you're saying!

If you *always* want two tags together, you can put them in the same bundle instead of in separate bundles.

<img width="366" src="https://user-images.githubusercontent.com/8336245/226206851-2357e0ac-f97e-4af6-99db-35ff6ba9b9aa.png">

If I had tags for... let's say, all of the Game of Thrones characters, and I wanted those tags to also include "game of thrones", I would make bundles like **title: anya** -> **tags: anya, game of thrones**.

Are you saying you'd like to be able to do this without taking up space in the bundles list? I.e. is the important part of your proposal

> me writing one tag would immediately make the others appear too

as opposed to having to click items from a big list? If so, I take it back, I guess, because then I do get what you're saying!

**Reply by hellersqueen** [reply]
yes exactly! finding the right bundle in a sea of bundles is getting harder and harder so i was kinda wondering if another option was possible

**Reply by marcustyphoon** [reply]
Ah, gotcha! So this is basically a "let me type instead of having to scroll and find the right thing and click on it" proposal. I'm always down for those, although the limitations of the user interface here might make it sort of tricky to implement!

**Reply by hellersqueen** [reply]
> "let me type instead of having to scroll and find the right thing and click on it" 

yeah that's a good way to put it lmao so following your previous game of thrones example, i'd like to just write the tag arya stark and have the game of thrones tag just append automatically following that. 

i'm trying to find any way possible to make my tagging process go faster and easier but i'm real bad at coding so i wanted to ask people better at it than me to confirm if that could be a possibility or not lol

**Comment by hellersqueen** [comment]
hey so I managed to make a little tampermonkey script that seems to kiiiiiinda work so far (even tho the coding is atrocious) so if no one else using xkit needed this thing then we can consider it resolved !
